internal/logout/logout.go

```diff
@@ -0,0 +1,34 @@
+package logout
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/spf13/afero"
+	"github.com/supabase/cli/internal/utils"
+)
+
+type RunParams struct {
+	Fsys afero.Fs
+}
+
+func PreRun() error {
+	if !utils.PromptYesNo("Do you want to log out, which will remove the token from your system?", true, os.Stdin) {
```

Comment:
Canceling removal should still be exitcode `0`, as it's terminating the program correctly. Not sure why project deletion is throwing error here.
However because `PreRun` cannot handle cancellation other than via error, you can move this check into `Run` and write

```go
if !utils.PromptYesNo("Do you want to log out, which will remove the token from your system?", true, os.Stdin) {
  fmt.Fprintln(stdout, "Not deleting token")
  return nil
}
```

Reply:
Thanks. Done.